front/app/containers/Admin/projects/edit/general/messages.ts

```diff
@@ -64,14 +64,14 @@ export default defineMessages({
     id: 'app.containers.AdminPage.ProjectEdit.timeline',
     defaultMessage: 'Timeline',
   },
-  areasLabel: {
+  areasLabelHint: {
     id: 'app.containers.AdminPage.ProjectEdit.areasLabel',
```

Comment:
The id needs to be changed, it's still the same for this one.

Reply:
Ah indeed. Thanks, let me try it.